components/server/src/user/user-controller.ts

```diff
@@ -141,6 +141,11 @@ export class UserController {
         router.get(
             "/login/ots/admin-user/:key",
             loginUserWithOts(async (req: express.Request, res: express.Response, user: User, secret: string) => {
+                // In case there is no/an empty key specified: Nobody should be able to call this so they are not able to guess values here
+                if (!this.config.admin.loginKey) {
+                    throw new ResponseError(500, "Cannot handle request");
```

Comment:
Can we make this nicer, like 
"No Admin login key configured, cannot login as superadmin"

Reply:
Done.